### PR TITLE
Velox

### PR DESCRIPTION
**Pull request by tmieslinger**
Hi, I would like to add an example code book chapter about VELOX IR imager cloud mask data.
The example is short, but combines the VELOX images with a dropsonde launch and shows the combination of the dataset with flight segment information and the JOANNE dataset.
Comments and further ideas are welcome :)

### CODE REVIEW

**Review by JuleRadtke — APPROVED** [review]
Very nice!! Good step by step explanations, I haven't anything to add.

File: how_to_eurec4a/velox.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# VELOX
+
+Video airbornE Longwave Observations within siX channels
+
+VELOX is a thermal infrared spectral imager (VELOX327k veL, 640 pixel by 512 pixels) with a synchronized filter wheel (at 100 Hz) covering six spectral channels in the thermal infrared wavelength range from 7.7 μm to 12.0 μm. The instrument measures the brightness temperature of upward radiance in a field-of-view of 35.49° by 28.71°. The PI during EUREC4A was Michael Schäfer (University Leipzig).
```
Comment (JuleRadtke) [inline how_to_eurec4a/velox.md:18]: Detailed description, I was just wondering, if it is too detailed, at least I don't know what a synchronized filter wheel is...?

File: how_to_eurec4a/velox.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                  )
+cbar = fig.colorbar(cax, ticks=ds.cloud_mask.flag_values)
+cbar.ax.set_yticklabels(ds.cloud_mask.flag_meanings.split(" "));
+
+ax.set_title(f"VELOX cloud mask at {sonde_dt}");
+```
+
+```{code-cell} ipython3
+print(f"Image minimum cloud fraction (most likely cloudy): {(ds_sel==2).sum().values / ds_sel.size:.2f}")
+print(f"Image maximum cloud fraction (most likely and probably cloudy) :{((ds_sel==1) | (ds_sel==2)).sum().values / ds_sel.size:.2f}")
```
Comment (d70-t) [inline how_to_eurec4a/velox.md:132]: These calculation do not handle undefined values correctly. I.e. `ds_sel.size` does count `unknown`-flagged entries.

Also the numeric values shouldn't be used. Just add something like:
```python
meanings = dict(zip(ds.cloud_mask.flag_meanings.split(" "), ds.cloud_mask.flag_values))
```
and use e.g. `meanings["most_likely_cloudy"]` in stead of `2`.

**Review by d70-t — COMMENTED** [review]
Looks good, I've just a few comments.

My general feeling was that the article covers maybe a bit too much of the dropsondes in relation to the amount which actually covers VELOX. But probably that's just fine.

File: how_to_eurec4a/velox.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+```{code-cell} ipython3
+ds_sel = ds.cloud_mask.sel(time=sonde_dt, method="nearest")
+```
+
+```{code-cell} ipython3
+fig, ax = plt.subplots()
+
+cax = ds_sel.plot(ax=ax,
+                  cmap=plt.get_cmap('Greys', lut=4),
```
[the remark this answers is not in the document]
Comment (tmieslinger) [reply how_to_eurec4a/velox.md:128]: The idea here was to keep the colors consistent with the time series plot below. So I would like to leave it though it's counter intuitive to have black clouds :)

File: how_to_eurec4a/velox.md (hunk 2 above)
Comment (tmieslinger) [reply how_to_eurec4a/velox.md:140]: I agree, that's a good point. I'll change that.

Comment (tmieslinger) [reply how_to_eurec4a/velox.md:140]: ...in the end it's a lot easier to take the `CF_min` and `CF_max` variables from the dataset and simply select the time. So I chaged the code to use those variables. 
However, the cloud frag values and meanings part will be covered in the cloud fraction chapter coming soon :-)

**Pull request merged**